src/MoBi.UI/Views/ParameterValuesView.cs

```diff
@@ -77,6 +85,15 @@ private void setParameterUnit(ParameterValueDTO parameterValue, Unit unit)
          });
       }
 
+      private void onGridViewMouseDown(MouseEventArgs e)
+      {
+         if (e.Button != MouseButtons.Right) return;
+         var location = new Point(e.X, e.Y + 50);
```

Comment:
We have to find the source of this `+ 50`. I did some digging around and came up with this little experiment. There are a configurable number of boxes that can be viewed here

![image](https://github.com/user-attachments/assets/858f3044-ab24-444e-a697-c829072b70ad)

In the ParameterValuesPresenter I arranged for them all to be shown by commenting out
```
         view.HideIsPresentView();
         view.HideRefreshView();
         view.HideNegativeValuesAllowedView();
```
I thought that the context menu would be shown in the right spot, I clicked in the very top/left corner of the grid and I saw that the problem was actually twice as bad now (top image none of the controls hidden, bottom with all but one hidden)!

![image](https://github.com/user-attachments/assets/4dd59fae-d7e1-4acf-9912-a4a2f999b568)

So, at least we know what is causing it. Can you do some more digging tomorrow to see what this could be? This will probably mean digging through some DevExpress documentation.

We can also discuss tomorrow.

Reply:
I think I know why this is happening.
There is a control of type: 
private DevExpress.XtraLayout.EmptySpaceItem emptySpaceItem;
which has a height of 50px:
this.emptySpaceItem.Size = new System.Drawing.Size(849, 50);

if I set this control to not visible with the method:
   public void HideSubPresenterGrouping()
   {
      emptySpaceItem.Visibility = LayoutVisibility.Never;
      layoutGroupPanel.GroupBordersVisible = false;
   }
then the menu is showing fine. Do you think this should be the solution?

Reply:
Does that hide the `Delete Values` controls?